deepomatic/cli/cmds/studio_helpers/file.py

```diff
@@ -6,7 +6,7 @@
 from .vulcan2studio import transform_json_from_vulcan_to_studio
 from ...thread_base import Greenlet
 from ...common import SUPPORTED_IMAGE_INPUT_FORMAT, SUPPORTED_VIDEO_INPUT_FORMAT
-from ...json_schema import is_valid_studio_json, is_valid_vulcan_json
+from ...json_schema import is_valid_studio_json, is_valid_vulcan_json, validate_json
```

Comment:
Are `is_valid_studio_json` and `is_valid_vulcan_json` still used? If not do we need them in `json_schema.py`?

```suggestion
from ...json_schema import validate_json
```